### ISSUE TITLE
Tumblr api changes break TumblThrees tumblr blog downloads.

### ISSUE THREAD

**Issue by Linktydraa** [johanneszab/TumblThree#222]
Hi Johannes,

While running downloader, I got this error saying something about a blog having a nonparsable post, and to change the posts per page in the details to 1. I've done that for all my blogs and even in the setting menu, but I still keep getting that message. Ive restarted the blogger several times as well.

**Comment by lesvannes** [comment]
Same here. When I hover, I get the following: 

> There was an error deserializing the object of type TumblThree.Applications.DataModels.TumblrApiJson.TumblrApiJson. Encountered unexpected character 'v'.

**Comment by chloeeisoaky** [comment]
same here

**Comment by johanneszab** [comment]
Yes, I've noticed that already as [mentioned here earlier](https://github.com/johanneszab/TumblThree/issues/221#issuecomment-382500301). 

Apparently Tumblr changed several of its apis, one official which isn't a nice thing to do without notice, and they also updated the svc api which they use internally for displaying dash board blogs. That is the interface we use to download hidden Tumblrs.

I've updated the code to reflect the changes. Let's see for how long they are good.
